streams/src/main/java/org/apache/kafka/streams/kstream/HoppingWindows.java

```diff
@@ -23,14 +23,16 @@
 import java.util.Map;
 
 /**
- * The hopping window specifications used for aggregations.
+ * The hopping window specifications used for aggregations. TODO extend
```

Comment:
I would not add all these `TODO` entries *within* the public java docs.

Rather, I'd add the entry after the docstring:

```java
/**
 * The hopping window [...]
*/
// TODO: extend the documentation
public class ...
```